src/data/blog.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "id": "liveness-2-and-beyond",
+    "name": "Liveness 2 and beyond",
+    "description": "",
+    "date": "11/17/23",
+    "tags": [],
+    "image": "https://defi-wonderland.notion.site/image/https%3A%2F%2Fprod-files-secure.s3.us-west-2.amazonaws.com%2F7683bccd-1174-4689-a817-b27fd9d7ef00%2F8e9b1568-aea1-478f-af39-74c39502a79d%2FDALLE_2023-11-01_15.45.54_-_Create_an_epic_and_dark_illustration_featuring_Alice_from_Alice_in_Wonderland_actively_engaged_in_the_process_of_synchronizing_two_different_worlds._.png?table=block&id=79955f2a-d4eb-49c6-b7fe-8b729f841958&spaceId=7683bccd-1174-4689-a817-b27fd9d7ef00&width=2000&userId=&cache=v2"
```

Comment:
should we use the image from notion or have it in another place hosted? @0xArdy @0xGorilla

Reply:
Yes, this should be done on another task tho